markets/perps-market/contracts/modules/LiquidationModule.sol

```diff
@@ -55,18 +55,51 @@ contract LiquidationModule is ILiquidationModule, IMarketEvents {
     /**
      * @inheritdoc ILiquidationModule
      */
-    function liquidateFlagged() external override returns (uint256 liquidationReward) {
+    function liquidateFlagged(
+        uint256 maxNumberOfAccounts
+    ) external override returns (uint256 liquidationReward) {
         uint256[] memory liquidatableAccounts = GlobalPerpsMarket
             .load()
             .liquidatableAccounts
             .values();
 
-        for (uint i = 0; i < liquidatableAccounts.length; i++) {
+        uint numberOfAccountsToLiquidate = maxNumberOfAccounts > liquidatableAccounts.length
+            ? liquidatableAccounts.length
+            : maxNumberOfAccounts;
+
+        for (uint i = 0; i < numberOfAccountsToLiquidate; i++) {
             uint128 accountId = liquidatableAccounts[i].to128();
             liquidationReward += _liquidateAccount(PerpsAccount.load(accountId));
         }
     }
```

Comment:
```suggestion
        for (uint i = 0; i < liquidatableAccounts.length; i++) {
            uint128 accountId = liquidatableAccounts[i].to128();
            liquidationReward += _liquidateAccount(PerpsAccount.load(accountId));
            
            maxNumberOfAccounts--;
            if (maxNumberOfAccounts == 0) break;
        }
    }
```

nit pik, just another approach

could also use `MathUtil.min` btw

Reply:
if there isn't any race condition you identify, I prefer mine approach (but using `MathUtil.min`) becase uses less gas.